src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonValueConverterOfT.cs

```diff
@@ -1,45 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Diagnostics.CodeAnalysis;
-
-namespace System.Text.Json.Serialization
-{
-    // Used for value converters that need to re-enter the serializer since it will support JsonPath
-    // and reference handling.
-    internal abstract class JsonValueConverter<T> : JsonConverter<T>
```

Comment:
This converter type may come back depending on work in https://github.com/dotnet/runtime/issues/36785.